ci/cpu/clang13_release_cxx20.yml

```diff
@@ -0,0 +1,30 @@
+include:
+  - local: 'ci/common-ci.yml'
+
+cpu clang13 release cxx20 deps:
+  extends: .build_deps_common
+  variables:
+    EXTRA_APTGET: "clang-13 libomp-13-dev"
+    COMPILER: clang@13.0.1
+    CXXSTD: 20
+    SPACK_ENVIRONMENT: ci/docker/release-cpu.yaml
+    USE_MKL: "ON"
+    BUILD_IMAGE: $CSCS_REGISTRY_PATH/release-cpu-clang13-20/build
+
+cpu clang13 release cxx20 build:
+  extends:
+    - .build_common
+    - .build_for_daint-mc
+  needs:
+    - cpu clang13 release cxx20 deps
+  variables:
+    DEPLOY_IMAGE: $CSCS_REGISTRY_PATH/release-cpu-clang13-20/deploy:$CI_COMMIT_SHA
+
+cpu clang13 release cxx20 test:
+  extends: .run_common
+  needs:
+    - cpu clang13 release cxx20 build
+  trigger:
+    include:
+      - artifact: pipeline.yml
+        job: cpu clang13 release cxx20 build
```

Comment:
The new line character is missing at the end of the file (see https://github.com/eth-cscs/DLA-Future/blob/master/misc/coding.md#vertical-spacing).
This is the problem reported by `check-format`

See https://github.com/eth-cscs/DLA-Future/actions/runs/6393725766/job/17353554378?pr=988#step:8:22

Other files has the same problem.